hurumap_ke/templates/profile/profile_detail.html

```diff
@@ -99,8 +99,6 @@ <h2 class="header-for-columns"><a class="permalink" href="#age" id="age">Age <i
             <div class="column-third">
                 {% include 'profile/_blocks/_stat_list.html' with stat=households.piped_water stat_type='percentage' %}
             </div>
-            <div class="column-two-thirds" id="chart-histogram-households-water_source_distribution" data-stat-type="scaled-percentage"
-                data-chart-title="Households by main source of water"></div>
```

Comment:
Why is this removed?

Reply:
@DavidLemayian this was removed accidentally. 😞